app/components/checkbox/checkbox.html.erb

```diff
@@ -1,7 +1,9 @@
+<input type='hidden' value='0' name="<%= name %>">
 <input
   type="checkbox"
   id="<%= id %>"
   name="<%= name %>"
-  <%= value ? 'checked' : '' %>
+  value='1'
+  <%= checked ? 'checked' : '' %>
   <%= required ? 'required' : '' %>
-/>
+/>
```

Comment:
If you haven’t yet installed something like that, you might want to check out an editorconfig plugin for your editor, e.g. this [plugin for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig). :)